packages/shared-integration/src/transformers.ts

```diff
@@ -40,7 +40,7 @@ export async function applyTransformers(
   }
 
   if (code !== original) {
-    ctx.affectedModules.add(id)
+    // ctx.affectedModules.add(id)
```

Comment:
It seems that this line of code is not needed. It will cause additional updates. For example, I modified file a, but multiple files may be added to the change list for the first time. Modification of file a will cause hot updates of all files. uno.css is also updated multiple times. If the a file changes, you should only need to initiate the hot update of the a file and the hot update of uno.css.